src/main/webapp/stories.html

```diff
@@ -0,0 +1,276 @@
+              <h4 class="pb-3 mb-4">Jahia</h4>
+              <p>Jahia brings together content management, data marketing and great integrations to help you deliver the best digital experiences.</p>
+              <a href="https://www.jahia.com/" target="_blank">https://www.jahia.com/</a>
+            </div>
+        </div>
+
+        <div class="row mb-5 mt-5">
+            <div class="col">
+              <h4 class="pb-3 mb-4">OpenHAB</h4>
+              <p>A vendor and technology agnostic open source automation software for your home.</p>
+              <a href="https://www.openhab.org/" target="_blank">https://www.openhab.org/</a>
+            </div>
+        </div>
+
+      </div><!-- Powered by Karaf -->
+
+      <div class="container">
+
+        <!-- Run on Karaf -->
+        <div class="row mb-5 mt-5">
+            <div class="col">
+              <h2 class="pb-3 mb-4 font-italic border-bottom"><i class="fas fa-server"></i> Run on Apache Karaf</h2>
+              <p>Software using Apache Karaf as runtime.</p>             
+            </div>
+        </div>
+
+        <div class="row mb-5 mt-5">
+            <div class="col">
+              <h4 class="pb-3 mb-4">Liferay</h4>
+              <p>Build portals, intranets, websites and connected experiences on the most flexible platform around.</p>
+              <a href="https://www.liferay.com/" target="_blank">https://www.liferay.com/</a>
+            </div>
+        </div>
+
+        <div class="row mb-5 mt-5">
+            <div class="col">
+              <h4 class="pb-3 mb-4">Open Studio for ESB</h4>
+              <p>Enterprise Service Bus from Talend.</p>
+              <a href="https://www.talend.com/products/application-integration/esb-open-studio/" target="_blank">https://www.talend.com/products/application-integration/esb-open-studio/</a>
+            </div>
+        </div>
+
+        <div class="row mb-5 mt-5">
+            <div class="col">
+              <h4 class="pb-3 mb-4">Fuse</h4>
+              <p>Enterprise Service Bus from RedHat.</p>
+              <a href="https://www.redhat.com/en/technologies/jboss-middleware/fuse/" target="_blank">https://www.redhat.com/en/technologies/jboss-middleware/fuse/</a>
+            </div>
+        </div>
+
+        <div class="row mb-5 mt-5">
+            <div class="col">
+              <h4 class="pb-3 mb-4">WebSphere</h4>
+              <p>J2EE application server from IBM.</p>
+              <a href="https://www.ibm.com/us-en/marketplace/java-ee-runtime/" target="_blank">https://www.ibm.com/us-en/marketplace/java-ee-runtime/</a>
+            </div>
+        </div>
+        
+      </div><!-- Run on Karaf -->
+
+      <div class="container">
+
+        <!-- Support Karaf -->
+        <div class="row mb-5 mt-5">
+            <div class="col">
+              <h2 class="pb-3 mb-4 font-italic border-bottom"><i class="fas fa-puzzle-piece"></i> Support Apache Karaf</h2>
+              <p>Frameworks that support Apache Karaf</p>             
```

Comment:
I'm not sure about this section as we will probably miss some frameworks. This section could be tricky to maintain especially for frameworks "outside" of Apache foundation. Thoughts ?

Reply:
You're right, I will remove it.